contracts/interfaces/ITrigger.sol

```diff
@@ -2,13 +2,13 @@ pragma solidity ^0.8.5;
 
 /**
  * @notice Abstract contract for creating or interacting with a Trigger contract
- * @dev All trigger contracts created must inerit from this contract and conform to this interface
+ * @dev All trigger contracts created must inherit from this contract and conform to this interface
  */
 abstract contract ITrigger {
-  /// @notice Trigger name, analgous to an ERC-20 token's name
+  /// @notice Trigger name, equivalent to an ERC-20 token's name
   string public name;
 
-  /// @notice Trigger symbol, analgous to an ERC-20 token's symbol
+  /// @notice Trigger symbol, equivalent to an ERC-20 token's symbol
```

Comment:
A bit of a nitpick, but I'm hesitant to use the work "equivalent" instead here, strictly because there are _not_ ERC-20 tokens, and so by definition it's not equivalent, but it does serve the same purpose.

Though it looks like there was a type in "analogous" which would need to be addressed anyway!

Reply:
Switched to "similar"